### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by kylepeeler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,5 +14,22 @@
...
+    "react-test-renderer": "^17.0.1"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|ogg)$": "<rootDir>/src/__mocks__/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/src/__mocks__/styleMock.js"
+    }
+  },
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.9"
```
Comment (mewelling) [inline package.json]: why is this not a devDependency?

File: src/__tests__/konami-code.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// __tests__/konami-code.js
+// these imports are something you'd normally configure Jest to import for you
+// automatically. Learn more in the setup docs: https://testing-library.com/docs/react-testing-library/setup#cleanup
+import '@testing-library/jest-dom'
+// NOTE: jest-dom adds handy assertions to Jest and is recommended, but not required
+
+import * as React from 'react'
```
Comment (mewelling) [inline src/__tests__/konami-code.js:7]: no longer needed if we update to react 17

File: package.json (hunk 1 above)
Comment (kylepeeler) [reply package.json]: Should be thanks

File: src/__tests__/konami-code.js (hunk 2 above)
Comment (kylepeeler) [reply src/__tests__/konami-code.js:7]: React isn't a direct dependency of this package, it's a peer dependency, thus I think it's best we keep this in? Will see what I can do in my next iteration when I bring in `tsdx`

**Pull request merged**